app/jobs/export_structural_job.rb

```diff
@@ -9,13 +9,9 @@ def perform(bulk_action_id, params)
     super
 
     CSV.open(csv_download_path, 'w', headers: true) do |csv|
-      csv << ['druid', *StructureSerializer::HEADERS]
+      csv << StructureSerializer::HEADERS
       with_items(params[:druids], name: 'Export structural metadata') do |cocina_object, success, failure|
-        rows_for_file = item_to_rows(cocina_object, success, failure)
-        druid = Druid.new(cocina_object).without_namespace
-        rows_for_file.each do |row|
-          csv << [druid, *row]
-        end
+        item_to_rows(cocina_object, success, failure).each { |row| csv << row }
       end
```

Comment:
lines 14-18 above can be collapsed into one line, since now we don't need to add the druid specifically (it will come from the single object export)